### DISCUSSION TITLE
Redstone controls

### DISCUSSION THREAD

**Discussion by Terrorbiest18** [MikaylaFischler/cc-mek-scada#413] [Support] [answered]
Hello everyone, I have understood everything about the controls, what I can't do is on the 1.20.1 the Redstone controls can maybe help me?

I want the line to open as soon as the Fissin reactor has too little coolant, but I can't handle the new config.

Thanks in advance

**Comment by MikaylaFischler** [comment, marked as answer]
It sounds like you might want to use the emergency coolant functionality, which is covered in this video: https://www.youtube.com/watch?v=XKit718Ucd8

That uses the old config file, the new configuration would be here: https://www.youtube.com/watch?v=8Rk860eRMH4

This enables a tube (or more than one) to either supply water into a reactor or water into a boiler, depending on your setup. This uses more conditions than just low coolant in the reactor, so it can catch problems quicker. Additionally, it'll keep flow continuous until the reactor is okay again.
<img width="610" alt="image" src="https://github.com/MikaylaFischler/cc-mek-scada/assets/2926271/653c65c9-9850-41d4-8a92-92e02305f0fb">

If you don't want to use this and instead just want a redstone control based on low coolant, you would configure this on the RTU and set it to be for that particular reactor unit. This won't activate until coolant level is <10%, but reactors will start uncontrollably overheating as it falls below ~27% fill. This redstone output is based on the level the reactor trips at, so the RPS would have activated and SCRAM'd the reactor at that point. This output was intended to activate things like an industrial alarm or redstone lamps, but it can be used for emergency coolant as well, but it lacks a lot of the smarts that the emergency coolant output has.
<img width="610" alt="image" src="https://github.com/MikaylaFischler/cc-mek-scada/assets/2926271/621e4333-b5fb-46ec-bfaa-bf9248d82ec0">

**Reply by Terrorbiest18** [reply]
Perfect, it wasn't actually a problem with the controls but with the Redstone receiver, which was set incorrectly.

Thank you very much for the quick response.
